src/components/Calculator.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+const buttons = ['AC', '+/-', '%', '÷', '7',
+  '8', '9', 'x', '4', '5', '6', '-',
+  '1', '2', '3', '+',
+  '0', '.', '-'];
+
+const MakeArray = () => {
+  const arrayForButtons = [];
+  buttons.forEach((button) => arrayForButtons.push(
+    <button className="calc-btns" type="button">{button}</button>,
+  ));
+  return arrayForButtons;
+};
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <>
+        <div id="calculator">
+          <div id="display-calculator" />
+          <input />
+        </div>
+        <div id="calculator-buttons">
+          <MakeArray />
+        </div>
+      </>
+    );
+  }
+}
```

Comment:
- [ ] As mentioned by the previous reviewer, Please make your calculator design match the given design. 

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/199251804-3c54908a-cf7f-484c-9f92-9de8eaf22ad6.png)  | ![image](https://user-images.githubusercontent.com/68796232/199251835-39460fc4-93c7-4ac6-b926-c8e0c40ff74e.png)  |